lib/main_common.pm

```diff
@@ -502,7 +502,10 @@ sub load_slepos_tests {
 sub load_docker_tests {
     loadtest "console/docker";
     loadtest "console/docker_runc";
-    if (is_sle('=12-SP3') || is_sle('=15')) {
+    if (is_sle('=15')) {
```

Comment:
I understand that point of course, those SLE docker images as well as the infrastructure around are still under development. The point why I submitted this PR is to have 'something' to see and eventually debug. It will be adjusted before going to the production.